Landing page updates and corresponding skill.md (#134)
charmed-hpc/docs: howto/manage/index.md

# Manage your cluster

> Perform day-to-day administration of a Charmed HPC cluster after initial deployment. These tasks cover both controller-level configuration and compute node operations.

Reviewer: I think we can shorten this sentence:

```suggestion
Administer a Charmed HPC cluster after initial deployment. These tasks cover both controller-level configuration and compute node operations.
```

"day-to-day" is a bit too marketing-speak for me 😅
